.github/workflows/autobuild.yml

```diff
@@ -69,7 +70,8 @@ jobs:
       build_all_targets:            ${{ steps.decide-build-targets.outputs.build_all_targets }}
     env:
       release_changelog_path:       ./.github_release_changelog.md
-
+    permissions:
+        contents:                   write
```

Comment:
Needed for release

Reply:
Is it worth putting comments like this in the yaml?

Reply:
Done.